lib/ranked-model.rb

```diff
@@ -24,14 +25,49 @@ def self.included base
       }
     end
 
+    rails_version = ActiveRecord.version.to_s
+    if rails_version < '5.0'
+      base.module_eval <<-EOS.strip_heredoc, __FILE__, __LINE__ + 1
+        def handle_ranking_return_value(return_value)
+          @ranked_model_ranking_succeeded
```

Comment:
some hoops that let us handle aborting the save, here no matter where in the ranker the problem originated, for the supported Rails versions